src/test/java/cloud/orbit/actors/cluster/impl/lettuce/LettuceMapTest.java

```diff
@@ -0,0 +1,280 @@
+ modification, are permitted provided that the following conditions
+ are met:
+
+ 1.  Redistributions of source code must retain the above copyright
+     notice, this list of conditions and the following disclaimer.
+ 2.  Redistributions in binary form must reproduce the above copyright
+     notice, this list of conditions and the following disclaimer in the
+     documentation and/or other materials provided with the distribution.
+ 3.  Neither the name of Electronic Arts, Inc. ("EA") nor the names of
+     its contributors may be used to endorse or promote products derived
+     from this software without specific prior written permission.
+
+ THIS SOFTWARE IS PROVIDED BY ELECTRONIC ARTS AND ITS CONTRIBUTORS "AS IS" AND ANY
+ EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ DISCLAIMED. IN NO EVENT SHALL ELECTRONIC ARTS OR ITS CONTRIBUTORS BE LIABLE FOR ANY
+ DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package cloud.orbit.actors.cluster.impl.lettuce;
+
+import org.junit.Assert;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import cloud.orbit.actors.cluster.NodeAddress;
+import cloud.orbit.actors.cluster.NodeAddressImpl;
+import cloud.orbit.actors.cluster.RedisClusterConfig;
+import cloud.orbit.actors.cluster.impl.RedisConnectionManager;
+import cloud.orbit.actors.cluster.impl.RedisKeyGenerator;
+import cloud.orbit.actors.cluster.impl.RedisShardedMap;
+import io.lettuce.core.ScriptOutputType;
+import io.lettuce.core.codec.ByteArrayCodec;
+import io.lettuce.core.codec.RedisCodec;
+import io.lettuce.core.codec.StringCodec;
+
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+
+public class LettuceMapTest
+{
+
+    RedisClusterConfig config;
+    RedisConnectionManager connectionManager;
+
+    FstObjectCodec codec = new FstObjectCodec();
+
+    boolean setup = false;
+    String clusterName = "testcluster";
+
+    @Before
+    public void setup() {
+        if (setup) {
+            return;
+        }
+
+        config = new RedisClusterConfig();
+        connectionManager = new RedisConnectionManager(config);
+        setup = true;
+    }
+
+    @Test
+    public void testClientsCreated() {
+        Assert.assertNotNull(config);
+
+        List<LettuceClient<String, Object>> clients = connectionManager.getActorDirectoryClients();
```

Comment:
`connectionManager.getActorDirectoryClients()` is repeated on line 85. I think one of these two lines was intended to check the `nodeDirectoryClients` instead.